Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -735,7 +735,7 @@ WebPageProxy::WebPageProxy(PageClient& pageClient, WebProcessProxy& process, Ref
 #if ENABLE(CONTEXT_MENUS)
     , m_contextMenuClient(makeUnique<API::ContextMenuClient>())
 #endif
-    , m_navigationState(makeUnique<WebNavigationState>())
+    , m_navigationState(makeUniqueWithoutRefCountedCheck<WebNavigationState>(*this))
```

Comment:
Should use `makeUniqueRef` and `m_navigationState` should be a `UniqueRef`, not a `std::unique_ptr`, since it can never be null.